extra-radio/libax25/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=libax25
+PKGSEC=hamradio
+PKGDES="Library for AX.25, ROSE, 6PACK, KISS and NETROM protocols"
+PKGDEP="gcc zlib"
+
+AUTOTOOLS_AFTER="--prefix=/usr \
```

Comment:
All of these are already defined in `AUTOTOOLS_DEF`.

Reply:
I removed these in other packages, but I forgot this package ...... Will fix soon.